CorsixTH/Lua/dialogs/patient.lua

```diff
@@ -158,12 +163,8 @@ function UIPatient:draw(canvas, x_, y_)
     end
   end
   -- How warm the patient feels
-  local warmth_bar_width = 22
-  local warmth = patient.attributes["warmth"]
-  if warmth then
-    warmth = self.normaliseWarmth(warmth)
-    warmth_bar_width = math_floor(warmth * 40 + 0.5)
-  end
+  local warmth = self.normaliseWarmth(patient.attributes["warmth"])
+  local warmth_bar_width = math_floor(warmth * 40 + 0.5)
```

Comment:
Another comment said a bar would be half-filled if it has an unknown value. I think 20 is close enough to 22 to make that hold.